### PR TITLE
Update WinUICs Default Template to use new System Backdrop APIs

### LINKED ISSUE #4641: Update Template Studio (WinUI) to use `SystemBackdrop = new MicaBackdrop();`
https://github.com/microsoft/WindowsAppSDK/issues/3408

Windows App SDK 1.3 Experimental 1: 1.3.230202101-experimental1 introduced `SystemBackdrop = new MicaBackdrop();`
 i. lieu of 

    <windowex:WindowEx.Backdrop>
        <windowex:MicaSystemBackdrop/>
    </windowex:WindowEx.Backdrop>

### LINKED ISSUE #4656: Please update to Windows App SDK 1.3
### Is your feature request related to a problem? Please describe.

More features available in new version, like
```cs
SystemBackdrop = new MicaBackdrop();
```
and more Window functions etc.

### Describe the solution you'd like

_No response_

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Applies to the following platforms:

- [X] WinUI
- [ ] WPF
- [ ] UWP

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
# PR checklist

**Quick summary of changes**

**Which issue does this PR relate to?**

**Applies to the following platforms:**

- [x] WinUI
- [ ] WPF
- [ ] UWP

Close #4656 & close #4641.

Need more manual tests to know if this changes work for everyone.

### CODE REVIEW

**Comment by Mysterious-Dev** [comment]
CIs fixed ^^

File: code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml [hunk 1]
```diff
@@ -10,7 +10,4 @@
     MinHeight="500"
     PersistenceId="MainWindow"
     mc:Ignorable="d">
-    <windowex:WindowEx.Backdrop>
```
Comment (mikebattista) [inline code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: Can we keep this defined in XAML like https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.media.micabackdrop?view=windows-app-sdk-1.3#examples shows?

**Review by mikebattista — CHANGES_REQUESTED** [review]

File: code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml (hunk 1 above)
Comment (mikebattista) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: Yes.

Comment (Mysterious-Dev) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: Don't think so :/

image
image

Comment (Mysterious-Dev) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: And with that

https://github.com/microsoft/microsoft-ui-xaml/pull/8139/files#diff-4f203bcadb6647af5b21c52a5d9464ca0c85fe096fd5cfd856626d53aa2e7882R99

Comment (Mysterious-Dev) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: And when I try to remove MicaKind, I have this result

image
image

Comment (mikebattista) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: @dotmorten have you tried this yet?

Comment (Mysterious-Dev) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: He said that.

https://github.com/microsoft/TemplateStudio/issues/4641#issuecomment-1516902916

Comment (dotMorten) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: Odd the doc shows this. It wasn’t working for me

Comment (Mysterious-Dev) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: So, we wait a fix ?

Comment (mikebattista) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: Yes. Looks like they're taking it through servicing so the fix should be released soon.

Comment (bpulliam) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: Servicing update 1.3.1 contains this fix and is rolling out as we speak. The nuget package is already available.

Comment (Mysterious-Dev) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: Nice 👍. Servicing update would be released before the end of the week?

Comment (Mysterious-Dev) [reply code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml:13]: I update the version of WASDK. It's done.

**Comment by Mysterious-Dev** [comment]
It's weird, why do I only have 4 tests even if I update the version of a dependency :/ ?

**Comment by Mysterious-Dev** [comment]
@mikebattista I reverted the update of WASDK and wait the merge of #4666 ^^.

**Review by bpulliam — APPROVED** [review]

**Comment by Mysterious-Dev** [comment]
Well, I think this PR can be merged with recent activity of #4666 ^^.

**Comment by pratikone** [comment]
looks good to be merged

**Review by pratikone — APPROVED** [review]

**Pull request merged**